pom.xml

```diff
@@ -52,6 +52,7 @@
     <spring.version>4.1.4.RELEASE</spring.version>
     <fcrepo.version>4.4.0</fcrepo.version>
     <fcrepo-transform.version>4.4.0</fcrepo-transform.version>
+    <fcrepo-java-client.version>4.4.1-SNAPSHOT</fcrepo-java-client.version>
```

Comment:
I thought about that... 

We're going to get this into jenkins so the snapshot releases exist in the maven repo, then I guess I should roll a non-snapshot release, then update this PR?  

Alternately, we could leave this as snapshot, but then when a release of fcrepo-camel is made, you'll be warned about snapshot dependencies by the release tooling and at that time, lock it down to a non-snapshot version.  We're certainly going to face times in the future when the snapshot version of fcrepo-camel *MUST* point to a snapshot version of fcrepo-java-client, or we'll have to roll releases every time we need to make a change in that client code for use by dependent applications.